llvm/test/CodeGen/AArch64/framelayout-sme-basepointer.mir

```diff
@@ -0,0 +1,28 @@
+# RUN: llc -mtriple=aarch64-none-linux-gnu -run-pass=prologepilog -mattr=+sme %s -o - | FileCheck %s --check-prefix=SME
+--- |
+    define void @hasBasepointer_sme_streaming() "aarch64_pstate_sm_enabled" { ret void }
+...
+---
+# This test verifies that the basepointer is available in presence of SME stack objects.
+# This is almost identical to the SVE version of this test but has to be a separate file
```

Comment:
Can you just mark the target attribute on the IR function, instead of using -mattr?  Not a big deal either way.